thoth/investigator/investigate_unresolved_package.py

```diff
@@ -137,8 +137,7 @@ def parse_unresolved_package_message(unresolved_package: MessageBase) -> None:
         revsolver_packages_seen: List[Tuple[str, str]] = []
 
         # Loop versions from the latest one
-        # TODO: Create method in thoth-python for sorting
-        for version in versions[::-1]:
+        for version in versions:
```

Comment:
These will either be of type `Version` or `LegacyVersion`, I'm pretty sure it was string before. I don't know exactly what side effects this will have.

They provide a `__str__` method so it you should be able to just cast it.

Reply:
Updated!